src/site/content/en/blog/learn-to-bundle/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Learn to bundle
+subhead: |
+    Learn to bundle your website.
+date: 2019-11-11
+hero: hero.png
```

Comment:
The hero image looks a bit blurry to me. Might want to check if it was run through compression. I'd suggest not compressing it if possible (our image CDN will handle that in production). Also recommend using [a 3200 x 960 size hero](https://web.dev/handbook/markup-media/#hero-images) if possible (that'll look nicer on 2x screens).